README.md

```diff
@@ -197,6 +204,41 @@ To uninstall the module, delete the following subtrees:
 * /apps/saas-aem-module-packages
 * /home/users/system/saas
 
+## Getting started
+
+After installation of the module following steps need to be done:
+
+### OSGI Configurations
+* Configure [Search as a Service - Search Api Request Executor Service Configuration](http://localhost:4502/system/console/configMgr/com.valtech.aem.saas.core.http.client.DefaultSearchApiRequestExecutorService):
+  * Base URL
+  * JWT Token
+* Configure [Search as a Service - Search Admin Request Executor Service Configuration](http://localhost:4502/system/console/configMgr/com.valtech.aem.saas.core.http.client.DefaultSearchAdminRequestExecutorService)
+  * Base URL
+  * JWT Token or Basic Authentication 
+
+Within AEM as a Cloud Service Base URL and JWT Token can be configured as a [secret environment variable](https://experienceleague.adobe.com/docs/experience-manager-cloud-service/content/implementing/deploying/configuring-osgi.html?lang=en#secret-configuration-values) 
+
+### Context-aware configurations
+
+The following fields are mandatory: SaaS Index
+
+### Sample content
+
+To get started right away you can install the sample [ui.content](https://github.com/valtech-ch/saas-aem-module/releases/latest) via CRX Package Manager and start [exploring](http://localhost:4502/editor.html/content/saas-aem-module/us/en/search-page.html).
+Optionally you can install the [wcm io Context-Aware Configuration Editor](https://wcm.io/caconfig/editor/usage.html) in order to see the context aware configuration within AEM.  
+
+### Embed component in custom templates
+
+Obviously you can embed the components into your own pages and templates.
+
+1. Open Template editor of the custom template
+2. Open Page Policies and 
+* include JS client library **saas-aem-module.base** in Client Libraries JavaScript Page Head
+* optionally add client library **saas-aem-module.wknd.sample** to have sample styling. This part you want to customize (see [Clientlibs](#clientlibs))
+3. Open Policy of the responsive grid and include component group Search as a Service - Content in allowed components list.
+
```

Comment:
For a rookie in AEM such as me, this would probably not yet be enough explanation on what I have to do in order to get started adding the components. Maybe someone with more experience can tell if this should be enough information :) 
Could also be tricky for Juniors, but again, you've got more experience than me here.

Reply:
it should be common knowledge but I can add few screenshots

Reply:
Thank you, that helps a lot :)